changelogs/fragments/1368-empty_bucket_policy.yml

```
@@ -0,0 +1,2 @@
+bugfix:
+- s3_bucket - deal gracefully with empty bucket policy (i.e. from wasabi cloud storage)"
```

```suggestion
- s3_bucket - deal gracefully with empty bucket policy (https://github.com/ansible-collections/amazon.aws/pull/1368).
```

Changelog entries should include a link to the issue or the PR.  This helps folks dig into the what and why of a change if it's potentially significant to them